dataRetriever/txpool/shardedTxPool.go

```diff
@@ -240,7 +241,13 @@ func (txPool *shardedTxPool) removeTx(txHash []byte, cacheID string) bool {
 
 // RemoveSetOfDataFromPool removes a bunch of transactions from the pool
 func (txPool *shardedTxPool) RemoveSetOfDataFromPool(keys [][]byte, cacheID string) {
+	timeStart := time.Now()
 	txPool.removeTxBulk(keys, cacheID)
+	duration := time.Since(timeStart)
+
+	if duration > 300*time.Millisecond {
+		log.Warn("RemoveSetOfDataFromPool", "duration", duration.Milliseconds())
```

Comment:
why not simplifying to:
```GO
log.Warn("RemoveSetOfDataFromPool", "duration", duration)
```
as this will automatically display the duration with the most appropriate time unit